### ISSUE TITLE
Accept `latest` as an argument for `install` and `use`

### ISSUE THREAD

**Issue by ardislu** [trailofbits/solc-select#138]
Currently, `solc-select install` only accepts a specific version number or the word `all`. `solc-select use` only accepts a specific version.

It would be convenient if these commands accepted `latest` as an alias for the latest version of Solidity. For example (assuming `0.8.17` is the latest version of Solidity):

```
$ solc-select install latest
Installing '0.8.17'...
Version '0.8.17' installed.
```

And:

```
$ solc-select use latest
Switched global version to 0.8.17
```

**Comment by CJ42** [comment]
I am also looking for this feature, as I am implementing this in a CI job that needs to check that the contracts compile to multiple versions including the latest one.

**Comment by elopez** [comment]
I'll close this issue as this feature got released in [v1.0.4](https://github.com/crytic/solc-select/releases/tag/v1.0.4)
